packages/core/src/components/date-dropdown/date-dropdown.tsx

```diff
@@ -163,15 +164,17 @@ export class DateDropdown {
    * This event is emitted when the date range changes within the component.
    * The event payload contains information about the selected date range.
    */
-  @Event() private dateRangeChange: EventEmitter<DateRangeChangeEvent>;
+  // eslint-disable-next-line @stencil-community/decorators-style
```

Comment:
Why?

Reply:
There are two conflicting rules in effect on this line (line length exceeded and decorator needs to be one liner).